daily_milestone.go

```diff
@@ -131,69 +129,63 @@ func getMilestones(baseURL string, token string, projectID string) ([]string, er
 	respByte, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
```

Comment:
You are checking for the error, but you wouldn't return an error to the calling function. You would actually try to return list at the end of the function even so the request might have an error.

Reply:
So should I return empty string and error right here ?

Reply:
Once here and once for the actual request above.

Reply:
Oh understood if it's an error the function should return changed it.